client/src/pages/locations/GeoLocation.js

```diff
@@ -113,25 +116,31 @@ const MGRSFormField = ({
   lat,
   lng,
   editable,
-  setValues,
+  setFieldValue,
   setFieldTouched,
   isSubmitting
 }) => {
   const [mgrs, setMgrs] = useState("")
 
-  useEffect(() => {
-    if (!editable && lat === null && lng === null) {
-      setMgrs("")
-    } else {
-      const mgrsValue = convertLatLngToMGRS(lat, lng)
-      if (mgrsValue) {
-        setMgrs(mgrsValue)
-        if (editable) {
-          setValues({ displayedCoordinate: mgrsValue, lat: lat, lng: lng })
+  // Coordinate should be sync between input field and map interaction
+  useEffect(
+    function syncMapEdit() {
+      if (!editable && lat === null && lng === null) {
+        setMgrs("")
+      } else {
+        const mgrsValue = convertLatLngToMGRS(lat, lng)
+        if (mgrsValue) {
+          setMgrs(mgrsValue)
+          if (editable) {
+            setFieldValue("displayedCoordinate", mgrsValue)
+            setFieldValue("lat", lat, false)
```

Comment:
These lat and lng setting seems unnecessary, we don't have any change that should trigger setting lat or lng.

Reply:
This was refactored, so not applicable anymore